tools/upgrade-to-1.1.0.sh

```diff
@@ -0,0 +1,212 @@
+    fi
+    echo "backup if there are any role defined for splunk operator"
+    kubectl get role splunk:operator:namespace-manager -n ${current_namespace} -o yaml >> ${backup_file_name}
+    if [ $? == 0 ] 
+    then 
+        echo "" >> ${backup_file_name}
+        echo "---" >> ${backup_file_name}
+    fi
+    echo "backup if there are any role-biding defined for splunk operator"
+    kubectl get rolebinding splunk:operator:namespace-manager -n ${current_namespace}  -o yaml >> ${backup_file_name}
+    if [ $? == 0 ] 
+    then 
+        echo "" >> ${backup_file_name}
+        echo "---" >> ${backup_file_name}
+    fi
+    echo "backup if there are any cluster-role defined for splunk operator"
+    kubectl get clusterrole splunk:operator:resource-manager -o yaml  >> ${backup_file_name}
+    if [ $? == 0 ] 
+    then 
+        echo "" >> ${backup_file_name}
+        echo "---" >> ${backup_file_name}
+    fi
+    echo "backup if there are any cluster-role-binding defined for splunk operator"
+    kubectl get clusterrolebinding splunk:operator:resource-manager -o yaml >> ${backup_file_name}
+    if [ $? == 0 ] 
+    then 
+        echo "" >> ${backup_file_name}
+        echo "---" >> ${backup_file_name}
+    fi
+    echo "backup deployment details"
+    kubectl get deployment splunk-operator -n ${current_namespace} -o yaml >> ${backup_file_name}
+    if [ $? == 0 ] 
+    then 
+        echo "" >> ${backup_file_name}
+        echo "---" >> ${backup_file_name}
+    fi
+    echo "--------------------------------------------------------------"
+    echo "backup of all the previsou splunk operator installation is complete, backup file is found in current diretory ${backup_file_name}"
+}
+
+delete_operator() {
+    echo "--------------------------------------------------------------"
+    echo "deleting all the previsous splunk operator resources....."
+    echo "deleting clusterrole"
+    kubectl delete clusterrole splunk:operator:resource-manager 
+    echo "deleting cluster rolebinding"
+    kubectl delete clusterrolebinding splunk:operator:resource-manager 
+    echo "deletign deployment"
+    kubectl delete deployment splunk-operator -n ${current_namespace}
+    echo "deleting serviceaccount"
+    kubectl delete serviceaccount splunk-operator -n ${current_namespace}
+    echo "deleting role"
+    kubectl delete role splunk:operator:namespace-manager -n ${current_namespace}
+    echo "deleting rolebinding"
+    kubectl delete rolebinding splunk:operator:namespace-manager -n ${current_namespace}
+    echo "--------------------------------------------------------------"
+    echo "previous instance of splunk operator removed"
+}
+
+deploy_operator() {
+    echo "--------------------------------------------------------------"
+    echo "installing splunk operator 1.1.0....." 
+    kubectl apply -f ${manifest_file}
+    echo "--------------------------------------------------------------"
+    echo "deployment new splunk opearator 1.1.0 complete"
```

Comment:
Do we need to look for the Running state, when confirming this?

Reply:
added code to wait for pod to come to ready state